libs/core/src/lib/multi-input/multi-input.component.ts

```diff
@@ -72,6 +74,12 @@ import { MULTI_INPUT_COMPONENT, MultiInputInterface } from './multi-input.interf
 export class MultiInputComponent
     implements MultiInputInterface, ControlValueAccessor, CssClassBuilder, OnInit, OnChanges, AfterViewInit, OnDestroy
 {
+    protected tokenCountHiddenLabel = `fdp-multi-input-token-count-id-${uniqueHiddenLabel++}`;
+
+    /** token  count hidden label */
+    @Input()
+    tokenHiddenId: string = this.tokenCountHiddenLabel;
```

Comment:
I think the property name is not descriptive enough, it's hard to get what this input is for

Reply:
fixed